hassio/src/dialogs/network/dialog-hassio-network.ts

```diff
@@ -129,75 +130,282 @@ export class DialogHassioNetwork extends LitElement
 
   private _renderTab() {
     return html` <div class="form container">
-        <ha-formfield label="DHCP">
-          <ha-radio
-            @change=${this._handleRadioValueChanged}
-            value="dhcp"
-            name="method"
-            ?checked=${this._device!.data.method === "dhcp"}
-          >
-          </ha-radio>
-        </ha-formfield>
-        <ha-formfield label="Static">
-          <ha-radio
-            @change=${this._handleRadioValueChanged}
-            value="static"
-            name="method"
-            ?checked=${this._device!.data.method === "static"}
-          >
-          </ha-radio>
-        </ha-formfield>
-        ${this._device!.data.method !== "dhcp"
-          ? html` <paper-input
+        ${IP_VERSIONS.map((version) =>
+          this._interface![version] ? this._renderIPConfiguration(version) : ""
+        )}
+        ${this._interface?.type === "wireless"
+          ? html`
+              <ha-expansion-panel outlined>
+                <span slot="title">Wi-Fi</span>
+                ${this._interface?.wifi?.ssid
+                  ? html`<p>Connected to: ${this._interface?.wifi?.ssid}</p>`
+                  : ""}
+                <mwc-button
+                  @click=${this._scanForAP}
+                  ?disabled=${this._scanning}
+                >
+                  ${this._scanning
+                    ? html`<ha-circular-progress active size="small">
+                      </ha-circular-progress>`
+                    : "Scan for accesspoints"}
+                </mwc-button>
+                ${this._accessPoints?.accesspoints.length !== 0
+                  ? this._accessPoints?.accesspoints
+                      .filter((ap) => ap.ssid)
+                      .map(
+                        (ap) =>
+                          html`
+                            <mwc-list-item
+                              twoline
+                              @click=${this._selectAP}
+                              ?activated=${ap.ssid ===
```

Comment:
Can this be a property?
```suggestion
                              .activated=${ap.ssid ===
```